wdl/imp_sub.wdl

```diff
@@ -178,9 +182,12 @@ task post_imputation {
         time Rscript --no-save /tools/r_scripts/plot_INFO_and_AF_for_imputed_chrs.R ${base} ${ref_panel_freq} ${chr} ${base}_varID_AF_INFO_GROUP.txt
 
         # annotate, edit tags and index
-        [[ ${base} =~ (.*)_qcd_imputed ]]
-        batch=${dollar}{BASH_REMATCH[1]}
-        edits="AF:AF_$batch INFO:INFO_$batch CHIP:CHIP_$batch AC_Het:AC_Het_$batch AC_Hom:AC_Hom_$batch HWE:HWE_$batch NS:NS_$batch AR2: DR2: IMP:"
+        edits="AR2: DR2: IMP:"
+        if ${add_batch_suffix}; then
+            [[ ${base} =~ (.*)_qcd_imputed ]]
```

Comment:
wondering what the point of this test is in here. I guess pipefail fails the whole thing in case this does not match?
would be better to be explicit if thats the case by checking and erroring out... could be also a remnant left over when deleting code.

Reply:
Yeah it's a weird one. Once I have time I can test if there's any real purpose to it.